dao/test/execute.test.mligo

```diff
@@ -0,0 +1,115 @@
+#import "tezos-ligo-fa2/test/helpers/list.mligo" "List_helper"
+#import "./helpers/token.mligo" "Token_helper"
+#import "./helpers/dao.mligo" "DAO_helper"
+#import "./helpers/suite.mligo" "Suite_helper"
+#import "./helpers/log.mligo" "Log"
+#import "./helpers/time.mligo" "Time_helper"
+#import "./helpers/assert.mligo" "Assert"
+#import "./bootstrap/bootstrap.mligo" "Bootstrap"
+#import "../src/main.mligo" "DAO"
+
+let () = Log.describe("[Execute] test suite")
+
+(* Boostrapping of the test environment, *)
+let init_tok_amount = 33n
+let bootstrap () = Bootstrap.boot(init_tok_amount)
+
+(* Successful timelock execution of an operation list *)
+let test_success =
+    let (tok, dao, sender_) = bootstrap() in
+    let dao_storage = Test.get_storage dao.taddr in
+
+    (* empty operation list *)
+    let () = Suite_helper.make_proposal_success(tok, dao, Some((
+        0xef67ec8260f062258375ab178c485146d467843d2a69b8eae7181441397f4021,
+        OperationList
+    ))) in
+    let () = Time_helper.advance(dao_storage.config.timelock_delay) in
+
+    let r = DAO_helper.execute(1n, 0x0502000000060320053d036d, dao.contr) in
```

Comment:
instead of hard coded bytes can you`Bytes.pack` the function as you did for  `test_success_parameter_changed`?

Reply:
I named all the packed and hash params